torch/_higher_order_ops/cond.py

```diff
@@ -233,7 +233,7 @@ def trace_cond(proxy_mode, func_overload, pred, true_fn, false_fn, operands):
         operands, (list, tuple)
     ), f"Cond operands must be a list or tuple of tensors and SymInts {operands}"
     assert all(
-        isinstance(o, (torch.Tensor, torch.SymInt)) for o in operands
+        isinstance(o, (torch.Tensor, torch.SymInt, int)) for o in operands
```

Comment:
why do we need ints now here?

Reply:
Yeah, we don't need them. Some invariants for wether a key need to handle int/symint: https://github.com/pytorch/pytorch/pull/138363#discussion_r1819402477

Reply:
I guess I was wrong: there's the case of running nested hops in eager, when top-level runs in eager, we make_fx the subgraph that contains the hop (with symint lifted args). The subgraph will receive real inputs. E.g. in a nested cond case the true_fn could be:
```
  4     def forward(self, arg0_1, arg1_1, arg2_1):
  5         sym_size_int = torch.ops.aten.sym_size.int(arg0_1, 0)
  6         sym_size_int_1 = torch.ops.aten.sym_size.int(arg0_1, 1)
  7         sym_size_int_2 = torch.ops.aten.sym_size.int(arg2_1, 0)
  8         true_graph_0 = self.true_graph_0
  9         false_graph_0 = self.false_graph_0
 10  ->     cond = torch.ops.higher_order.cond(arg1_1, true_graph_0, false_graph_0, [arg0_1, arg2_1, sym_size_int, sym_size_int_1, sym_size_int_2]);  arg1_1 = true_graph_0 = false_graph_0 = arg0_1 = arg2_1 = sym_size_int = sym_size_int_1 = sym_size_int_2 = None
 11         getitem = cond[0];  cond = None
 12         return [getitem]
```
, where sym_size_int and sym_size_int_1 are concrete integers.